pkg/api/fnresult/v1/types.go

```diff
@@ -29,6 +29,8 @@ type Result struct {
 	// If user provides an executable file with commands, ExecPath should
 	// contain the entire input string.
 	ExecPath string `yaml:"exec,omitempty"`
+
+	Wasm string `yaml:"wasm,omitempty"`
```

Comment:
Since this is going to be in alpha, we don't need to resolve this now. But I'm wondering if having a new field for each potential runtime might not be the best syntax. I wondering if it would be better to have a dedicated `runtime` field to better organize these details (I think this is similar to what is proposed in `Catalog`).

Reply:
Removed it and reusing image and exec now.